### ISSUE TITLE
Helm init fails on fresh install

### ISSUE THREAD

**Issue by l0rd** [che-incubator/chectl#27]
It looks like the step fails even if `helm init` is successful (maybe helm init doesn't return zero even if the command is successful?). 

Anyway the workaround is to rerun `chectl server:start`

```log
Error: Command failed: helm init --service-account tiller --wait
$HELM_HOME has been configured at ~/.helm.
Tiller (the Helm server-side component) has been installed into your Kubernetes Cluster.
Please note: by default, Tiller is deployed with an insecure 'allow unauthenticated users' policy.
To prevent this, run helm init with the --tiller-tls-verify flag.
For more information on securing your installation see: https://docs.helm.sh/using_helm/#securing-your-helm-installation
    at makeError (/snapshot/chectl/node_modules/execa/index.js:174:9)
    at module.exports.Promise.all.then.arr (/snapshot/chectl/node_modules/execa/index.js:278:16)
```

**Comment by azatsarynnyy** [comment]
I'm no longer faced with that problem anymore. Although, I'm using chectl really often on fresh minikube installation last two weeks.
It looks weird, taking into account that nothing has been changed in chectl regarding that command since last month.

**Comment by l0rd** [comment]
I have also tried to reproduce it last week but failed to do so.

**Comment by l0rd** [comment]
Should be fixed by #54
